### PR TITLE
feature(decision-date-expired) - Added the decision date expired page and fixed missing govuk assets

### PR DESCRIPTION
**Pull request by jmorgan74**
## Changes

- Added path to express for 'assets'.
- Created "decision date expired" view template.
- Added jest and an eligibility controller unit test (with dummy passing test which needs to be implemented).

### CODE REVIEW

File: services/forms-web-app/.prettierrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "singleQuote": true,
   "printWidth": 100,
-  "trailingComma": "all"
+  "trailingComma": "none"
```
Comment (MrSimonEmms) [inline services/forms-web-app/.prettierrc:4]: How come we've changed this?

**Review by MrSimonEmms — COMMENTED** [review]
The image looks ok but not sure on changes to prettier - don't object as such, but the trailing comma does make reading `git diff` and PRs easier

File: services/forms-web-app/.prettierrc (hunk 1 above)
Comment (jmorgan74) [reply services/forms-web-app/.prettierrc:4]: I was having issues with this line:

```
app.use(
  '/assets',
  express.static(path.join(__dirname, '../node_modules/govuk-frontend/govuk/assets'))
);
```

Comment (MrSimonEmms) [reply services/forms-web-app/.prettierrc:4]: Oh right, that would have been adding a trailing to the function - can you try `trailingComma: "es5"` and see if that works https://prettier.io/docs/en/options.html#trailing-commas

I'd prefer to keep it in so it's in the AirBnB style guide

Comment (jmorgan74) [reply services/forms-web-app/.prettierrc]: lovely sir! done! 👍

**Review by MrSimonEmms — APPROVED** [review]
Glad to help

**Pull request merged**